.github/workflows/on-main-branch-commit-push-liminal-node-to-ecr.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Builds and pushes aleph-node liminal to ECR on push to main branch
+
+on:
+  push:
+    branches:
+      - main
+      # temp for testing
+      - add-test-liminal-binary-to-workflows
```

Comment:
Did it work?

Reply:
After some more changes, but eventually yes: https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/4498596592